spec/unit/puppet_pal_catalog_spec.rb

```diff
@@ -96,6 +96,21 @@
           expect(result).to eq('worked1')
         end
 
+        it 'instantiates a user defined resource definition in the given code string' do
+          result = Puppet::Pal.in_tmp_environment('pal_env', modulepath: modulepath, facts: node_facts) do |pal|
+            pal.with_catalog_compiler do |compiler|
+              compiler.evaluate_string(<<-CODE)
```

Comment:
I don't see any tests for the new function. Perhaps this was intended to parse the string and call `evaluate` instead of just calling `evaluate_string`?